README.md

```diff
@@ -1,153 +1,277 @@
-# vinyl-fs [![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Coveralls Status][coveralls-image]][coveralls-url] [![Dependency Status][depstat-image]][depstat-url]
-## Information
-<table>
-  <tr><td>Package</td><td>vinyl-fs</td></tr>
-  <tr><td>Description</td><td>Vinyl adapter for the file system</td></tr>
-  <tr><td>Node Version</td><td>>= 0.10</td></tr>
-</table>
+<p align="center">
```

Comment:
For consistency, this is being added to all gulpjs projects to indicate it follows our eslint/jscs, testing standards and exists in the gulpjs org